tests/acl_config_test.py

```diff
@@ -36,7 +34,7 @@ def test_parse_table_with_vlan_and_duplicates(self):
         assert table_info["stage"] == "egress"
 
         # Since Ethernet4 is a member of Vlan1000 we should not include it twice in the output
-        port_list = table_info["ports@"].split(",")
+        port_list = set(table_info["ports"])
```

Comment:
>port_list [](start = 8, length = 9)

port_set?

Reply:
I mean the variable name is not proper. Not a blocking issue, but you can make it better readable.

---
In reply to: [598998736](https://github.com/Azure/sonic-utilities/pull/1519#discussion_r598998736) [](ancestors = 598998736)